src/main/java/me/eccentric_nz/TARDIS/commands/admin/TARDISDeleteCommand.java

```diff
@@ -75,19 +75,6 @@ boolean deleteTARDIS(CommandSender sender, String[] args) {
         HashMap<String, Object> where = new HashMap<>();
         Player player = null;
         if (tmp == -1) {
-            // this should be run from the console if the player running it is the player to be deleted
-            if (sender instanceof Player) {
-                player = (Player) sender;
-                if (player.getName().equals(args[1])) {
-                    HashMap<String, Object> wherep = new HashMap<>();
-                    wherep.put("uuid", player.getUniqueId().toString());
-                    ResultSetTravellers rst = new ResultSetTravellers(plugin, wherep, false);
-                    if (rst.resultSet()) {
-                        TARDISMessage.send(sender, "TARDIS_DELETE_NO");
-                        return true;
-                    }
-                }
-            }
```

Comment:
Technically I could now delete my own TARDIS while inside it...

Reply:
Correct. It seems to work perfectly fine from my testing, you get teleported out the instant the interior deletes and the police box starts dematerializing. Here's a short clip of how it looks from the inside and outside: https://tkno.pw/3q0fju0

Reply:
Okay cool